packages/api/src/utils/github/buildProjectsQuery.ts

```diff
@@ -0,0 +1,56 @@
+import { Project } from '../../entities/Project';
+import { env } from '../../env';
+import logger from '../../logger';
+
+interface Repository {
+  id: string;
+  nameWithOwner: string;
+}
+
+interface RepositoryResponse {
+  data: {
+    nodes: Repository[];
+  };
+}
+
+export const buildProjectsQuery = async (projects: Project[]) => {
+  const repoNodeIds: String[] = [];
+  projects.map((project) => repoNodeIds.push(project.nodeID));
```

Comment:
`map` actually returns an array, so you can just do this:
```suggestion
  const repoNodeIds = projects.map((project) => project.nodeID);
```